jobs/integr8ly/ocp3/nightly/osd-testing-nightly-trigger.yaml

```diff
@@ -0,0 +1,217 @@
+                                string(name: 'clientSecret', value: "${SSO_SECRET}")
+                            ]).result
+                        }
+                    } // stage
+
+                    if (installationStatus != 'SUCCESS') {
+                        println 'There were some errors in previous installation pipeline!'
+                        currentBuild.result = 'FAILURE'
+                        return
+                    }
+
+                    // This relies on a cronjob (or something else) to update the client secret in master-config.yaml so that we can successfully login afterwards
+                    stage ('Try to login to installed cluster') {
+                        println "Waiting for successful login to the ${CLUSTER_NAME} cluster"
+                        i = 0
+                        // Trying to log into the cluster 12 times and each 5 minutes which can take max 1 hour 
+                        while(!loggedIn && i < 12) {
+                            loggedIn = ocLogin()
+                            i += 1
+                            sleep time: 5, unit: 'MINUTES'
+                        }
+                    }
+
+                    // we want to run test only when there was a successful login to the cluster
+                    if (loggedIn) {
+
+                        stage ('Create admin user in SSO') {
+                            build(job: 'sso-user-create-tests', propagate: false, parameters: [
+                                string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                                string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                                string(name: 'CUSTOMER_ADMIN_USERNAME', value: "customer-admin"),
+                                string(name: 'CUSTOMER_ADMIN_PASSWORD', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+                                string(name: 'SSO_URL', value: "https://sso-openshift-sso.${ROUTER_SHARD}.${CLUSTER_NAME}.openshiftapps.com"),
+                                string(name: 'USERNAME', value: "admin"),
+                                string(name: 'NAMESPACE_PREFIX', value: "openshift-"),
+                                string(name: 'NUMBER_OF_USERS', value: "1"),
+                                booleanParam(name: 'CLEAN_RESOURCES', value:  Boolean.valueOf("false"))
+                            ])
+                        } // stage
+
+                        stage ('Create test user in SSO') {
+                            build(job: 'create-rhmi-user', propagate: false, parameters: [
+                                string(name: 'REPOSITORY', value: "${TEST_SUITES_REPOSITORY}"),
+                                string(name: 'BRANCH', value: "${TEST_SUITES_BRANCH}"),
+                                string(name: 'CLUSTER_URL', value: "https://console.${CLUSTER_NAME}.openshift.com"),
+                                string(name: 'ADMIN_USERNAME', value: "${ADMIN_USERNAME}"),
```

Comment:
`01` missing here :) (I know you know)

Reply:
actually no, it is defined as admin01 above

Reply:
It's not AFAIK - https://github.com/integr8ly/ci-cd/pull/307/files#diff-5450a374ad8dd23c37ee56c49958b893R30